velox/functions/prestosql/tests/JsonCastTest.cpp

```diff
@@ -1202,6 +1202,19 @@ TEST_F(JsonCastTest, castInTry) {
 }
 
 TEST_F(JsonCastTest, tryCastFromJson) {
+  auto test =
+      [&](const auto& toType, const auto& input, const auto& expectedOutput) {
+        auto castExpr = buildCastExpr(JSON(), toType, true);
+        exec::ExprSet exprSet({castExpr}, &execCtx_);
+        exec::EvalCtx context(&execCtx_, &exprSet, input.get());
+
+        std::vector<VectorPtr> result(1);
+        SelectivityVector rows(input->size());
+        exprSet.eval(rows, context, result);
+        test::assertEqualVectors(expectedOutput, result[0]);
+        ASSERT_FALSE(context.errors());
```

Comment:
It might be worth moving this to evaluateAndVerify or making evaluateAndVerifyTryCast it's own function and moving it there.  In general, it makes sense to check that context.errors() is empty.

Reply:
I see like for all runs we do check that. let me check if its easy